dom-event-delegation/main.js

```diff
@@ -0,0 +1,11 @@
+var $ul = document.querySelector('.task-list');
+$ul.addEventListener('click', function (event) {
+  console.log('event.target: ', event.target);
+  console.log('event.target.tagName: ', event.target.tagName);
+  if (event.target.tagName === 'BUTTON') {
+    var closestElement = event.target.closest('.task-list-item');
+    console.log('closest .task-list-item: ', closestElement);
+    var $li = document.querySelector('.task-list-item');
+    $li.remove(closestElement);
```

Comment:
🤔 You already have the `closestElement`, just remove that without querying the DOM again.